rubocop.yml

```diff
@@ -54,6 +54,7 @@ Layout/LineLength:
   IgnoreCopDirectives: false
   AllowedPatterns:
   - "\\A\\s*(remote_)?test(_\\w+)?\\s.*(do|->)(\\s|\\Z)"
+  - "(\\A|\\s)#"
```

Comment:
is it to allow a line followed by a long comment?

Reply:
@xiaoronglv yes, as we are going to add column comments in the future https://github.com/helloworld1812/workstream-backend/pull/4989, then the annotation for the model will also have comments, which can be pretty long for some fields.

Reply:
@teddy1004 imho, we can allow a long line with only comments, like the first code snippet, but not the second one. I'm good if this is the allowed pattern.

```ruby
def func
  # This is a long long long long long long long long long long long long long long long long long long long long long comment
  puts "This func is ready to be called"
end
```

```ruby
def func
   puts "This func is ready to be called" # This is a long long long long long long long long long long long long long long long long long long long long long comment
end
```

Reply:
@thrillertom yes, you are right, we would allow the second one.

the regular expression will match line start with `#` or spaces with `#`.